Modules/@babylonjs/react-native-iosandroid/android/src/main/java/com/babylonreactnative/EngineView.java

```diff
@@ -67,34 +70,58 @@ public void surfaceDestroyed(SurfaceHolder holder) {
     public void setAntiAliasing(Integer value) {
         BabylonNativeInterop.updateMSAA(value);
     }
+
+    public void setTopMost(Boolean topMost) {
+        setIsTransparentAndTopMost(this.isTransparent, topMost);
+    }
     // ------------------------------------
     // TextureView related
-
     public void setIsTransparent(Boolean isTransparent) {
+        setIsTransparentAndTopMost(isTransparent, this.topMost);
+    }
+
+    public void setIsTransparentAndTopMost(Boolean isTransparent, Boolean topMost) {
```

Comment:
This function is only used within this class, right? Not part of the contract I think?

```suggestion
    private void setIsTransparentAndTopMost(Boolean isTransparent, Boolean topMost) {
```